doc/nrf/app_dev/bootloaders_dfu/sysbuild_image_ids.rst

```diff
@@ -0,0 +1,107 @@
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``SB_CONFIG_MCUBOOT_ADDITIONAL_UPDATEABLE_IMAGES``     | Number of additional MCUboot images to include in MCUboot (see :ref:`sysbuild_assigned_images_ids_mcuboot_only`)                                     |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``SB_CONFIG_MCUBOOT_APP_SYNC_UPDATEABLE_IMAGES``       | If enabled, this option sets the number of MCUboot images for both MCUboot and the main application.                                                 |
+|                                                        | If disabled, it sets the number only for the MCUboot image.                                                                                          |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+.. _sysbuild_assigned_images_ids_mcuboot_only:
+
+MCUboot-only images
+*******************
+
+Some images are visible only to MCUboot, allowing it to manage them independently of the main application.
+This setup supports features like shared secondary slots.
+Currently, MCUboot uses this capability to update itself, which can be enabled with secure boot.
+MCUboot then shares the main application's secondary slot for firmware updates.
+However, it directly transfers the update to the designated MCUboot image slot, which is then booted by |NSIB|.
+
+The main application does not need to be aware of this process; it only needs to handle an MCUboot update as it would any normal application update and reboot to apply changes.
+This functionality is possible because there are additional MCUboot-only images that are not accessible to the main application and are configured only within MCUboot.
+
+The total number of images configured in MCUboot is the sum of ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES`` and ``SB_CONFIG_MCUBOOT_ADDITIONAL_UPDATEABLE_IMAGES``.
+If ``SB_CONFIG_MCUBOOT_APP_SYNC_UPDATEABLE_IMAGES`` is enabled, the main application sets only ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES``.
+Otherwise, it is not set.
+
+Getting image IDs in an application
+***********************************
+
+When building with sysbuild, the image IDs are provided to the application through Kconfig values and can be used in the code as needed.
+These image IDs are also used in |NCS| hooks for tasks such as nRF5340 network core updates, so no additional configuration is required.
```

Comment:
Is this an allusion to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf53/serial_recovery.html ? Can we link there?
```suggestion
These image IDs are also used in |NCS| hooks for various tasks (such as :ref:`nRF5340 network core updates <ug_nrf5340_serial_recovery>`), so no additional configuration is required.
```
or maybe to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf53/simultaneous_multi_image_dfu_nrf5340.html ? (In such case, the ref is `ug_nrf5340_multi_image_dfu`.)

Reply:
Fixed to second link